draft-ietf-scitt-architecture.md

```diff
@@ -544,12 +549,12 @@ Multiple Issuers can make the same Statement about a single Artifact, affirming
 
 ### Registration
 
-The same Signed Statement may be independently registered in multiple Transparency Services.
 To register a Signed Statement, the Transparency Service performs the following steps:
 
-1. **Client authentication:** This is implementation-specific and MAY be unrelated to the Issuer identity.
-Signed Statements may be registered by a different party than their Issuer.
-1. **Issuer Verification:** The Transparency Service MUST perform resolution of the Issuer's identity.
+1. **Client authentication:** A Client authenticates with the Transparency Service, to Register Signed Statements.
+Authentication and authorization is implementation-specific, and out of scope of the SCITT Architecture.
+Signed Statements may be registered by a different Client identity than the Issuer.
+1. **Issuer Verification:** The Transparency Service MUST perform resolution of the Issuer's identity, which may be different than the Client identity.
```

Comment:
This is registration policy details.... and interacts with all the other requirements we have for registration policies.... starting to feel like its better to say nothing here.